File: javascript/8kyu/Removing Elements.js
// Take an array and remove every second element from the array. Always keep the first element and start removing with the next element.

// Example:

// ["Keep", "Remove", "Keep", "Remove", "Keep", ...] --> ["Keep", "Keep", "Keep", ...]

// None of the arrays will be empty, so you don't have to worry about that!

// https://www.codewars.com/kata/5769b3802ae6f8e4890009d2

function removeEveryOther(arr){
    //your code here
    let newArray = [];
    for (let i = 0; i < arr.length; i += 2){
        newArray.push(arr[i]);
    }
    return newArray;
}

File: java/8kyu/Removing Elements.java
// Take an array and remove every second element from the array. Always keep the first element and start removing with the next element.

// https://www.codewars.com/kata/5769b3802ae6f8e4890009d2

public class Kata {
  
  public static Object[] removeEveryOther(Object[] arr) {
    Object[] output = new Object[(arr.length + 1) / 2];
    
    for (int i = 0; i < output.length; i++) {
        output[i] = arr[i * 2];
    }
    
    return output;
  }
}